test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -2410,13 +2442,23 @@ public void testMetadataColumns_blobNull() throws Exception {
 
   @Test
   public void testMetadataColumns_array() throws Exception {
-    assumeTrue("ARRAY type not supported", is(H2));
-    String[] content = { "hello", "world" };
-    executeUpdate("create table data(id int, content array)");
-    String sql = "insert into data(id, content) values (1, ?)";
-    PreparedStatement ps = prepareStatement(sql);
-    ps.setObject(1, content);
-    assertEquals(1, ps.executeUpdate());
+    if (is(H2)) {
+      String[] content = {"hello", "world"};
+      executeUpdate("create table data(id int, content array)");
+      String sql = "insert into data(id, content) values (1, ?)";
+      try (PreparedStatement ps = getConnection().prepareStatement(sql)) {
```

Comment:
Brett deleted this try statement on master, and the merge missed that. Restore this block to indentation-only changes.